test/test_sparse_csr.py

```diff
@@ -1256,6 +1259,32 @@ def test_direct_coo_csr_conversion(self, device, dtype):
 
             self.assertEqual(coo_sparse.to_sparse_csr().to_sparse_coo(), coo_sparse)
 
+    @ops(sparse_csr_linalg_solve)
```

Comment:
This test should skip CUDA (`@onlyCPU`) and this test should skip CPU if CUDA is not available.

Reply:
Done!